src/components/Context.js

```diff
@@ -46,13 +56,35 @@ const reducer = (action) => (state, props) => {
     };
   };
 
+  const loadItemsFromLocalStorage = () => {
+    return {
```

Comment:
JSON.parse throws an Exception if the data is corrupted. It's very unlikely that our data gets corrupted because of our mis-handling it, but we read stuff from local storage which can be easily altered by the users in their browsers. As a good practise - let's wrap JSON.parse in a try-catch.

How about that?

```
const readValue = key => {
  const value = localStorage.getItem(key);
  if (value) {
    try {
      return JSON.parse(value);
    } catch (e) {}
  }
}
```

and then: `cart: readValue("cart") || []`